Modules/timemodule.c

```diff
@@ -776,27 +776,100 @@ the C library strftime function.\n"
 #endif
 
 static PyObject *
-time_strftime(PyObject *module, PyObject *args)
+strftime1(time_char **outbuf, size_t *bufsize,
```

Comment:
nitpick: you might rename the function to `time_strftime1()` to avoid any conflict with a potential `strftime1()` name from the C library.